### ISSUE TITLE
ERR_CONNECTION_REFUSED after update

### ISSUE THREAD

**Issue by MiraMaX166** [rdugan/iceriver-oc#91]
I updated version 080 to 081 for ks0pro. After that I lost access to the device, the browser writes ERR_CONNECTION_REFUSED. I returned the firmware to default. Did a full reset. I tried to flash 081 again, but the result was the same - ERR_CONNECTION_REFUSED. I reset everything again and tried to flash 080, but now this firmware also gives ERR_CONNECTION_REFUSED. That is, now I can’t install any firmware other than the standard one or the firmware from Swift. What could be the matter? This is not an isolated case, all three of my devices are now suffering from this problem.

**Comment by MiraMaX166** [comment]
Version 071 just installed without problems

**Comment by rdugan** [comment]
081 uses https

https://github.com/rdugan/iceriver-oc#new-authauth-routines

https://github.com/rdugan/iceriver-oc#known-issues

**Comment by MiraMaX166** [comment]
Yes, I am aware that HTTPS is now used, but this is a different matter, since I waited about 30 minutes but could not log in from any browser (the cache and cookies were cleared through the browser extension).

**Comment by UHeinz13** [comment]
I have 5 KS0 Ultra, and one of the 5 miners has exactly the same problem. If I want to log in, I get to the main page, but when I click on the "Miner" menu item, I always get the message "Connection refused" and I end up back at the login window. It works problem-free with the 4 other miners. Tried everything possible, cleared cache, different browser, different PC, reset several times and re-flashed. Always the same behavior. Only Iceriver original firmware works at this unit.

**Comment by rdugan** [comment]
@UHeinz13 these do not sound like the same issue.  Your's is from some change you made to your pool setup that corrupted the database some how.  Just ssh into the machine and delete the /var/order/db directory (iceriver / 12345678, port 54188)

**Comment by UHeinz13** [comment]
I deleted the two files ir.bak and ir.mdb, restarted and now it works. Many many thanks

**Comment by bowuesser8107** [comment]
> @UHeinz13 these do not sound like the same issue.  Your's is from some change you made to your pool setup that corrupted the database some how.  Just ssh into the machine and delete the /var/order/db directory (iceriver / 12345678, port 54188)

Thanx for this tip.... 👍

**Comment by AndyBochum** [comment]
hello! is it the same by ks0 pro ? iceriver and 12345678

**Comment by rdugan** [comment]
@AndyBochum no, pros have the TCF port open so people were able to add ssh accessible accounts on their own by going through this process: https://github.com/incidrthreat/iceriver_oc/blob/main/ECLIPSE.md
